dvc/scm/git.py

```diff
@@ -173,3 +173,7 @@ def _install_hook(self, name, cmd):
     def install(self):
         self._install_hook("post-checkout", "checkout")
         self._install_hook("pre-commit", "status")
+
+    @classmethod
+    def belongs_to_scm(cls, path):
+        return os.path.basename(path) == cls.GITIGNORE or cls.GIT_DIR in path
```

Comment:
Would probably use `ignore_file` and `dir` properties instead. Especially since they are scm agnostic. Also, note that `cls.GIT_DIR in path` is error prone. Consider `.git-foo` case. You need a better approach 🙂